tests/test-dirs/hidden-deps/dash-h.t

```diff
@@ -90,7 +90,7 @@ the type)
           "line": 4,
           "col": 12
         },
-        "type": "sig type t = int val x : int @@ portable type s = Baz end",
```

Comment:
@goldfirere Please verify that this change is reasonable. I think this is right, but the `statefulness` axis is new to me.

Reply:
I've made a ticket to track this.